files/SZWEEXEC/ZWEGEN00

```diff
@@ -381,6 +388,55 @@ if card.0 > 0 then do
   say 'The job card was added to each member.'
 end
 
+
+
+say 'Checking the ZWESLSTC CONFIG entry'
+
+if AllocateDataSet(jclCopy'(ZWESLSTC)', 'zwejcl') > 0 then do
+  ExitWithRC(8)
+end
+if ReadFromDataSet('zwejcl') > 0 then do
+  ExitWithRC(8)
+end
+if FreeByDD('zwejcl') > 0 then do
+  ExitWithRC(8)
+end
+do j = 1 to !contentToRead.0
+  if COMPARE(contentToRead.j, 'CONFIG={ZWE_CLI_PARAMETER_CONFIG}') = 0 then do
+    say 'Defining CONFIG entry for ZWESLSTC'
+    currentline = j
+    ReadFromDataSet('myconfig')
```

Comment:
Cannot be standalone, it is returning a value, which is interpreted. 
It has to be `if ReadFromDataSet('myconfig') = 0 then do...`